### PR TITLE
fix: finding a suitable global executables directory

### PR DESCRIPTION
**Pull request by zkochan**
ref #2649

### CODE REVIEW

**Comment by zkochan** [comment]
The failing test is unrelated.

File: .changeset/strong-plants-shake.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+"@pnpm/config": minor
+"@pnpm/plugin-commands-installation": patch
+---
+
+A new setting is returned by `@pnpm/config`: `npmGlobalBinDir`.
```
Comment (aparajita) [inline .changeset/strong-plants-shake.md:6]: Does this mean that everyone for whom this is currently failing will have to manually specify the global bin dir?

Comment (aparajita) [inline .changeset/strong-plants-shake.md:6]: Does this mean that everyone for whom this is currently failing will have to manually specify the global bin dir?

Comment (zkochan) [reply .changeset/strong-plants-shake.md:6]: no. That directory is set by npm.

Comment (aparajita) [reply .changeset/strong-plants-shake.md:6]: Okay. See my comment in #2649 for what will fix that issue.

**Pull request merged**